release/models/types/openconfig-yang-types.yang

```diff
@@ -32,7 +32,13 @@ module openconfig-yang-types {
     Section 4.c of the IETF Trust's Legal Provisions Relating
     to IETF Documents (http://trustee.ietf.org/license-info).";
 
-  oc-ext:openconfig-version "0.3.1";
+  oc-ext:openconfig-version "1.0.0";
+
+  revision "2023-12-07" {
+    description
+      "Add hex-string-colon typedef";
+    reference "0.3.1";
```

Comment:
You forgot to update this version. Little surprised the CI didn't catch that 0.3.1 is duplicated.